src/analytics/internal/analytics.js

```diff
@@ -0,0 +1,153 @@
+import { MapEventTypeToInt, MapNotifTypeToIntArray } from './constants'
+import SendToServer from './sendToServer'
+import { remoteFunction } from 'src/util/webextensionRPC'
+import AnalyticsLogStorage from './AnalyticsLogStorage'
```

Comment:
Neither `AnalyticsLogStorage` or `SendToServer` are classes, so they shouldn't have *PascalCase*. Also the `incrementvalue` method in the `Analytics` class should be in *lowerCamelCase*`

Also their modules don't follow the naming conventions we use in the rest of the code-base; if there's multiple words in a module name, we use *kebab-case*.js (unless it's a UI module containing React components, then we use *PascalCase*.js).